### PR TITLE
Update dependencies version

### PR DESCRIPTION
**Pull request by Arhell**
**What type of PR is this?**

/kind cleanup
/kind feature

**Which issue(s) this PR fixes**:
Fixes https://github.com/karmada-io/website/issues/79

### CODE REVIEW

**Comment by Arhell** [comment]
https://deploy-preview-141--karmada.netlify.app/

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[build.environment]
+    NODE_VERSION = "16"
```
Comment (RainbowMango) [inline netlify.toml:2]: This isn't the auto-generated file, right?
If yes, please add a new line to the end.

Comment (Arhell) [reply netlify.toml]: No, I added it myself, I also added a line

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve
Thanks @Arhell

**Pull request merged**